src/alloc.rs

```diff
@@ -396,16 +365,18 @@ impl AllocMetadata {
         ptr_info
     }
 
+    /// Updates the metadata associated with an allocation. If the block was
```

Comment:
"If the block was moved" sounds a bit vague. I think this comment just means "Update the allocator's information so that whe block once pointed to by `ptr` is recorded as having moved to `new_ptr` and is now `size` bytes big." or similar?

Reply:
Fixed: 7b1ea0